pytorch_sphinx_theme/layout.html

```diff
@@ -94,11 +94,25 @@
           </li>
 
           <li>
-            <a href="{{ theme_variables.external_urls['features'] }}">Features</a>
+            <div class="ecosystem-dropdown">
```

Comment:
This section seems to be causing the render issues cause dropdown wasn't allowed before